src/utl/test/test_report_metrics.tcl

```diff
@@ -0,0 +1,4 @@
+read_lef ./../../../test/Nangate45/Nangate45.lef
+read_verilog ./../../../test/gcd_nangate45.v
+link_design gcd
+report_metrics 2 final
```

Comment:
You should pass `-metrics_report_dir results` to put the result file in the expected location.  You should also have an .rptok file and use diff_files to compare the report.